eng/BuildSamples.ps1

```diff
@@ -0,0 +1,5 @@
+﻿$baseDir = Join-Path $env:BUILD_SOURCESDIRECTORY "samples"

```

Comment:
Why do we need this script? If you want to find all the samples folders I'd suggest you do it in https://github.com/Azure/azure-sdk-for-net/blob/master/eng/service.proj like the other cases.

Reply:
I thought that was for service specific directories. This is for the root samples directory.